packages/react-ui-components/src/stories/Components/Breadcrumb.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { EBreadcrumbItemTarget, KvBreadcrumb } from '../../components';
+
+export default {
+	title: "Components/Breadcrumb",
+	component: 'kv-breadcrumb',
+	parameters: {
+		notes: require('../../../../ui-components/src/components/kv-breadcrumb/readme.md')
```

Comment:
Probably is better to add an alias for this path. 

I suggest to create an alias in main.js file.
```
const path = require("path");

module.exports = {
   ...,
   webpackFinal: async (config) => {
		config.resolve.alias = {
			...config.resolve?.alias,
			"@ui-notes": path.resolve(
				__dirname,
				"../../ui-components/src/components"
			)
		};
		return config;
	}
};
```

And on this file, is now possible to do something like this:
`notes: require('@ui-notes/kv-breadcrumb/readme.md')`